### ISSUE TITLE
Kismet error message

### ISSUE THREAD

**Issue by vasmane** [alphafox02/antsdr_dji_droneid#1]
I tried your repo, but keep getting kismet error message :
FATAL: Datasource helper(bladerf-wiphy) failed, could not process incoming control packet.
Im running latest dragon os with kismet nightly release and antsdr e200

**Comment by alphafox02** [comment]
I’m not sure if the nightlies include kismet_cap_antsdr_droneid yet. You’d have to manually compile if not. Are you able to run this command?

**Comment by alphafox02** [comment]
Looks like it’s now available 

https://www.kismetwireless.net/repos/apt/git/jammy/pool/main/k/kismet-capture-antsdr-droneid/

The error you’re referring to won’t affect the droneid capture tool. The important thing is you have another terminal window open and running the command in the readme and get a successful connection to the AntSDR. Successfully decoded DroneID will then populate within Kismet.

**Comment by vasmane** [comment]
Im not able to execute the command. Even after including the repo. Still command not found

**Comment by alphafox02** [comment]
All kismet packages have to be updated to the latest nightly, well technically just kismet and the kismet core packages. You’re almost there. Sounds like you installed the package I linked to above, not just make sure you’re running nightlies.

**Comment by vasmane** [comment]
Yep! Got it running , thanks a lot. It looks like it decodes ocusync protocols, because i've tried with lightbridge drone and it isnt there :/
